contrib/cirrus/runner.sh

```diff
@@ -119,6 +119,7 @@ exec_container() {
     exec podman run --rm --privileged --net=host --cgroupns=host \
         -v /dev/fuse:/dev/fuse \
         -v "$GOPATH:$GOPATH:Z" \
+        -v /usr/libexec/podman:/usr/libexec/podman \
```

Comment:
This isn't right.  For the `TEST_ENVIRON=container` case, we basically treat the container like a VM.  Very little should leak into the container from the host.